modules/imaging_browser/js/ImagePanel.js

```diff
@@ -482,32 +482,57 @@ ImageQCCommentsButton = React.createClass({
         );
     }
 });
-ImageDownloadButtons = React.createClass({
-    displayName: 'ImageDownloadButtons',
 
-    render: function () {
-        return React.createElement(
-            'div',
-            { className: 'row mri-second-row-panel col-xs-12' },
-            React.createElement(ImageQCCommentsButton, { FileID: this.props.FileID,
-                BaseURL: this.props.BaseURL
-            }),
-            React.createElement(DownloadButton, { FileName: this.props.Fullname,
-                Label: 'Download Minc',
-                BaseURL: this.props.BaseURL
-            }),
-            React.createElement(DownloadButton, { FileName: this.props.XMLProtocol,
-                BaseURL: this.props.BaseURL,
-                Label: 'Download XML Protocol'
-            }),
-            React.createElement(DownloadButton, { FileName: this.props.XMLReport,
-                BaseURL: this.props.BaseURL,
-                Label: 'Download XML Report'
-            }),
-            React.createElement(DownloadButton, { FileName: this.props.NrrdFile,
-                BaseURL: this.props.BaseURL,
-                Label: 'Download NRRD'
-            })
+LongitudinalViewButton = React.createClass({displayName: "LongitudinalViewButton",
```

Comment:
This should be done in the JSX file and compiled into the JS file. Doing it directly will likely cause it to get lost next time someone changes the file.